t/ui/02-csrf.t

```diff
@@ -70,7 +71,9 @@ $t->post_ok('/api/v1/jobs/99928/cancel' => form => {csrf_token => $token})->stat
 # test restart with and without CSRF token
 $t->post_ok('/api/v1/jobs/99928/restart' => form => {csrf_token => 'foobar'})->status_is(403);
 $t->post_ok('/api/v1/jobs/99928/restart' => {'X-CSRF-Token' => $token} => form => {})->status_is(200);
-$t->post_ok('/api/v1/jobs/99928/restart' => form => {csrf_token => $token})->status_is(200);
+# TODO why is this warning acceptable?
+my $warning = qr/Use of uninitialized value \$array_type in numeric eq/;
```

Comment:
catching the warning has two benefits
* it doesn't spill on the console
* **if** by any chance behaviour changes, the test can fail and the test developer can check if the change is acceptable

You can easily still show the warning when calling e.g. `prove -v <test_name>` or just `perl <test_name>`